src/components/VBtnToggle/VBtnToggle.js

```diff
@@ -73,6 +74,29 @@ export default {
       }
 
       this.$emit('change', items)
+    },
+    updateAllValues () {
+      if (!this.multiple) {
+        return
+      }
+
+      const items = []
+
+      for (let i = 0; i < this.buttons.length; ++i) {
+        const item = this.getValue(i)
+        const index = this.inputValue.indexOf(item)
+        if (index !== -1) {
+          items.push(item)
+        }
+      }
+
+      this.$emit('change', items)
+    }
+  },
+
+  created () {
+    if (this.multiple && !Array.isArray(this.inputValue)) {
+      consoleWarn('v-btn-toggle model must be bound to an array if the multiple property is true.')
```

Comment:
I will have to check, but I think we have a default way of showing components in the error messages, such as:

```consoleWarn('`v-btn-toggle` model must be...')```

Reply:
Yes, pass `this` as a second arg of `consoleWarn` and it will show the name of the component